.gitignore

```diff
@@ -1,4 +1,10 @@
-generator/
+generator/mock
```

Comment:
Can we change the directory where things are generated instead to be like "generator/generated" ?